index.html

```diff
@@ -1,1642 +1,256 @@
-		</tbody>
-	  </table>
+  <section id="sec-declaration">
+    <h2>Component Declaration</h2>
+    <p></p>
+    <div class="ednote">
+      Component interface (properties)
+    </div>
+  </section>
 
+  <section id="sec-reuse">
+    <h2>Component Reuse</h2>
+    <p></p>
+    <div class="ednote">
+      How to instantiate a component. Passing arguments, attributes. Access to component nodes.
+    </div>
+  </section>
+
+  <section id="sec-properties">
+    <h2>Component Properties and Methods</h2>
+    <p></p>
+    <div class="ednote">
+    </div>
+  </section>
+
+  <section id="sec-events">
+    <h2>Component Events</h2>
+    <section id="sec-events-binding">
+      <h3>Event Binding</h3>
+      <p></p>
+      <div class="ednote">
+      </div>
+    </section>
+    <section id="sec-events-listeners">
+      <h3>Event Listeners</h3>
+      <p></p>
+      <div class="ednote">
+        How
+      </div>
     </section>
-	<section>
-        <h2>list-item</h2>
-        <p>The 'list-item' is a child component of 'list' and is used to display items in a list.</p>
-        <p>The 'list-item' component has no additional attributes except the common attributes.</p>
-		<p>The 'list-item' component not support additional events except the common events.</p> 
+    <section id="sec-events-types">
+      <h3>Types of Events</h3>
+      <p></p>
+      <div class="ednote">
+        The common events for MiniApps.
+      </div>
     </section>
-    <section>
-        <h2>swiper</h2>
-        <p>The 'swiper' component provides a swiper container that enables the switch of child components.</p>
-		<p>In addition to common attributes, the 'swiper' component supports the following attributes:</p>
+  </section>
+
+  <section id="sec-scripting">
+    <h2>Scripting Resources</h2>
+    <p></p>
+    <div class="ednote">
+      Restrictions in the scripting resources.
+    </div>
+  </section>
+
+  <section id="sec-stylesheets">
+    <h2>Styling</h2>
+    <p>
+      MiniApp user agents support CSS profiles based on the standard specifications defined by the CSS Working Group Snapshot [[CSS-SNAPSHOT]]. Style sheets of MiniApp components follow the CSS recommendations for syntax, selectors, rules, media queries, fonts, and other elements described in [[CSS-SNAPSHOT]] to specify the presentation of the components.
+    </p> 
+    <p>
+      MiniApp user agents are not expected to implement full support of CSS. Instead, they MAY define specific CSS profiles with subsets of CSS properties, selectors, colors, rules, and other CSS specifications, aiming to simplify the development process and meet all their concrete requirements. 
+    </p>  
+    <p>
+      Every MiniApp component has a style sheet (with the `.css` extension) associated by default. Developers MAY include additional style sheets to reuse common styles among the components (e.g., styles for buttons or dialogs). The shared style sheets SHOULD be hosted in the `/Common` directory in the package.

```

Comment:
Should it be `/common` (lowercase)?

Reply:
Done: 8194f26f034e30ac67fa1c250c8e61b0af1b65f5
Thanks, @xfq